docs/v1/v1.0.rst

```diff
@@ -13,7 +13,9 @@ interpreted as described in `RFC 2119
 Status
 ------
 
-This specification is deprecated. See :ref:`zarr-specs` for the latest version.
+This specification is deprecated. See `v3
+<https://zarr-specs.readthedocs.io/en/latest/v3/core/v3.0.html>`_ for the
```

Comment:
with this change, we will need to update the 1.0 spec again if we ever release a version after 3

Reply:
Should we remove the link for v3 from v1 and v2, then?

Reply:
It is true forever for both v1 and v2 that they have been superseded, but it doesn't make sense to hard-code the latest version of the spec in the old spec documents. I think the correct thing is to say "there's a new version [link to list of zarr spec versions]", which I think is what was supposed to happen from the start